src/corppa/poetry_detection/annotation/command_recipes.py

```diff
@@ -0,0 +1,87 @@
+from collections import Counter, defaultdict
```

Comment:
Would be helpful to add a docstring either at the top or with the recipe explaining how you run this and showing some sample output.

Reply:
Thanks, that's a good idea. Although the styling itself is a bit outside of my understanding (I believe it's rendered by an external library)